index.bs

```diff
@@ -2019,23 +2025,29 @@ interface XRInputSourceEvent : Event {
   constructor(DOMString type, XRInputSourceEventInit eventInitDict);
   [SameObject] readonly attribute XRFrame frame;
   [SameObject] readonly attribute XRInputSource inputSource;
+  readonly attribute double? screenX;
+  readonly attribute double? screenY;
```

Comment:
What kind of logic do we expect apps to write with `screenX`/`screenY`?  We already give the app a `targetRaySpace` for raycasting into the 3D scene, and so the primary scenario I see here is activating a 2D button that the app rendered in `"viewer"` space.  However, apps that do that will not be usable on AR headsets.

Instead, we are solving the screen-space button problem in a cross-form-factor manner with [DOM overlays](https://immersive-web.github.io/dom-overlays/).  DOM overlays handle 2D input raycasting for you, and critically, they support headsets by allowing those overlays to be rendered on non-head-locked UI when using headsets.

Does this change undo that by assisting apps in directly rendering 2D content in `"viewer"` space?

Reply:
I would expect apps to us [XRInteractionMode](https://immersive-web.github.io/webxr-ar-module/#xrinteractionmode-enum) to do custom UI instead here. I can imagine an app using gamepads in world-space and input clicks on GL-rendered buttons in screen-space.

This does assist apps in directly rendering 2D content to the viewer space. I'm not sure if that really undoes this, since not everything supports DOM overlay, and I'd like there to be support for custom UI patterns that are not picked by the user-agent.